native/python/pyjp_object.cpp

```diff
@@ -239,14 +231,52 @@ static PyType_Spec objectSpec = {
 	objectSlots
 };
 
+static PyObject *PyJPException_new(PyTypeObject *type, PyObject *pyargs, PyObject *kwargs)
```

Comment:
Special constructors are needed.  There was a flaw in the old design in which the wrong new method was called.  We zero the memory so it wasn't causing an issue today, but I picked it up as an error when I extended as type in the sql work and got really unexpected results.  It should now be resolved.